doc/man7/EVP_PKEY-ML-KEM.pod

```diff
@@ -121,12 +132,29 @@ An B<ML-KEM-768> key can be generated like this:
 
 Equivalent calls are available for B<ML-KEM-512> and B<ML-KEM-1024>.
 
+An B<ML-KEM> private key in seed format can be converted to a key in expanded
+FIPS 203 B<dk> format by running:
+
+    $ openssl pkey -provparam pkey_retain_seed=no -in seed.pem -out long.pem
+
+To generate an, e.g., B<ML-KEM-768> key, in expanded format, you can run:
```

Comment:
drop the e.g here.. This is a stated list of examples.

Reply:
Maybe show the other way also?

via a -pkeyopt

Reply:
There is no pkeyopt that can do that, those apply during key generation, but we're doing decoding.